addons/sale/wizard/payment_link_wizard_views.xml

```diff
@@ -11,4 +11,25 @@
         <field name="binding_view_types">form</field>
     </record>
 
+    <record id="payment_link_wizard_view_form_inherit" model="ir.ui.view">
+        <field name="name">payment.link.wizard.form</field>
+        <field name="model">payment.link.wizard</field>
+        <field name="inherit_id" ref="payment.payment_link_wizard_view_form"/>
+        <field name="arch" type="xml">
+            <field name="amount" position="after">
+                <field name="show_confirmation_message" invisible="1"/>
+                <field name="amount_paid" readonly="1" attrs="{ 'invisible': [('amount_paid', '&lt;=', 0)]}"/>
+            </field>
+            <div class="alert alert-warning text-center fw-bold"
+                 attrs="{'invisible': [('is_amount_allowed', '=', True)]}"
+                 position="after">
```

Comment:
Please add a name in the targeted div and use the name to target it.
It's confusing here.